### DISCUSSION TITLE
Not found plots

### DISCUSSION THREAD

**Discussion by ccmaya** [Chia-Network/chia-blockchain#4804] [Q&A]
Can an expert help me? I used the plots drawn by MAC, but they could not be recognized by WIN2012, suggesting that the plots could not be found.
<img width="1036" alt="截屏2021-05-1318 58 00" src="https://user-images.githubusercontent.com/60649225/118116890-71ba0600-b41d-11eb-8954-f20ce5b3435b.png">

**Comment by lvcivs** [comment]
I assume you used the same keys?

Have you used `chia plots check` to see whether the plots are OK?

Have you tried re-adding the plots dir in the GUI?
